Clarify example snippet based upon feedback (#5341)
DataDog/documentation: content/en/tracing/faq/resource-trace-doesn-t-show-up-under-correct-service.md

An example of modifying the top level name for Python can be found below:

```
   @tracer.wrap('tornado.notify', 
                service='tornado-notification', 
                resource='MainHandler.do_something')
    @tornado.gen.coroutine
    def do_something(self):
        # do something
```

This function explicitly sets both the service name and Top Level Name, being `tornado-notification` and `tornado.notify` respectively.

> Also note that we set our resource name manually, `MainHandler.do_something`. 

Reviewer: ```suggestion
Also note that the resource name is set manually, `MainHandler.do_something`. 
```

> By default our resource name would be set to this as it's the name of the function and the class for which it lives under in Tornado.

Reviewer: ```suggestion
By default the resource name would be set to this as it's the name of the function and the class for which it lives under in Tornado.
```
